pkgs/development/python-modules/cloudsmith-api/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, certifi
+, six
+, dateutil
+, urllib3
+}:
+
+buildPythonPackage rec {
+  pname = "cloudsmith-api";
+  version = "0.54.15";
+
+  format = "wheel";
```

Comment:
Would you be open to using the upstream git repo? https://github.com/cloudsmith-io/cloudsmith-api/tree/master/bindings/python/src .

Generally wheels are discouraged because we wouldn't have tests, and if there are any compiled extensions, it's usually easier to just build it from source than trying to patchelf everything.

Reply:
In this case, it looks like the tests are just empty classes, probably because the library is just auto-generated from OpenAPI schema.

Reply:
Ah, then nevermind